README.md

```diff
@@ -17,4 +17,5 @@ Current devices in the medical industry are usually not accessible by doctors du
 A difference to existing projects like OpenBCI is that it will not be necessary to 3D print large headsets. Instead we are focusing on creating a device that collects as much data as possible through the forehead. To achieve this goal we are using high-grade sensors and flexible electronics.
 
 # External Dependencies
-*preference fix library* : com.takisoft.fix:preference-v7:$rootProject.prefFixLibraryVersion"
+* *preference fix library* : com.takisoft.fix:preference-v7:$rootProject.prefFixLibraryVersion"
+* *Library for providing the audio-device interface* : [JSyn Library for Android](http://www.softsynth.com/jsyn/beta/jsyn_on_android.php) | [github repo reference](https://github.com/iamareebjamal/jsyn).
```

Comment:
No need to add all of these in README, we can see them in build.gradle

Reply:
Shall I provide the website link or leave that as well of the audio library.